app/components/paperShow/components/collectionBox.scss

```diff
@@ -0,0 +1,375 @@
+@import "../../../variables";
+
+$color-mono-00: #1F2A36;
+$color-mono-01: #34495e;
+$color-mono-03: #77828C;
+$color-mono-04: #9AA7B5;
+$color-mono-06: #DAE1E8;
+$color-mono-07: #F0F1F2;
+$color-mono-08: #F7F9FA;
+$color-mono-09: #FAFBFC;
+$color-mono-10: #FDFEFF;
+
+$color-blue-01: #133F7A;
+$color-blue-03: #3e7fff;
+$color-blue-07: #E6EEFF;
+
+
+.fab {
+  position: fixed;
+  bottom: 24px;
+  right: 24px;
+  z-index: 100;
+
+  * {
+    margin: 0;
+    padding: 0;
+    font-family: "Roboto", sans-serif;
```

Comment:
is this necessary part? we already use `font-family: "Roboto", "sans-serif";` for `body` element and this element could inherit that font-family.